src/Compilers/CSharp/Test/Symbol/Symbols/GenericConstraintTests.cs

```diff
@@ -6862,5 +6862,190 @@ static void Main(string[] args)
                 Diagnostic(ErrorCode.ERR_BadTypeArgument, "R2").WithArguments("int*").WithLocation(6, 7)
                 );
         }
+
+        [Fact]
+        [WorkItem(41779, "https://github.com/dotnet/roslyn/issues/41779")]
+        public void Bug41779_Original()
+        {
+            var source =
+@"interface I
+{
+    object GetService();
```

Comment:
>object GetService(); [](start = 4, length = 20)

Is presence of this API necessary for the scenario?

Reply:
The API allows the program to call the GetService generic method, which leads to the error, from what I can tell.